src/Compilers/CSharp/Test/Emit/CodeGen/FixedSizeBufferTests.cs

```diff
@@ -298,5 +298,119 @@ unsafe struct S
                 //     fixed int F[3, 4];
                 Diagnostic(ErrorCode.ERR_FixedBufferTooManyDimensions, "[3, 4]"));
         }
+
+        [Fact, WorkItem(1171076, "DevDiv")]
+        public void UIntFixedBuffer_01()
+        {
+            var text =
+@"
+using System;
+using System.Runtime.InteropServices;
+
+[StructLayout( LayoutKind.Sequential )]
+public unsafe struct AssemblyRecord
+{
+    public fixed byte Marker[ 8 ];
+    public fixed UInt32 StartOfTables[ 16 ];
+}
+
+class Program
+{
+    static unsafe void Main( string[ ] args )
+    {
+        UInt32 [] arr = new UInt32[18];
+        for (int i = 0; i < arr.Length; i++)
+        {
+            arr[i] = (uint)(120 + i);
+        } 
```

Comment:
It looks like we compile this code but don't verify the IL in the test.  Is there a reason we need to keep this portion?